src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -42,6 +45,37 @@ const TeamMembers = createReactClass({
     }
   },
 
+  removeMember(member) {
+    let {params} = this.props;
+    leaveTeam(
+      this.api,
+      {
+        orgId: params.orgId,
+        teamId: params.teamId,
+        memberId: member.id,
+      },
+      {
+        success: () => {
+          this.setState({
+            memberList: this.state.memberList.filter(m => {
```

Comment:
does this api request return something you can use to do the filtering? not a big deal

Reply:
it returns the team, so nothing really to help with filtering